src/nnet3/nnet-utils.h

```diff
@@ -283,6 +283,9 @@ void CollapseModel(const CollapseModelConfig &config,
        Note: this sets the 'underlying' learning rate, i.e. it will get
        multiplied by any 'learning-rate-factor' set in the nodes.
 
+    set-learning-rate-factor [name=<name-pattern>] learning-rate-factor=<learning-rate-factor>
+       Sets the learning rate factor for any updatable nodes matching the name pattern.
```

Comment:
nodes -> components

Reply:
I agree! Thanks, also probably I should change this issue for set-learning-rate. Do you agree?